### ISSUE TITLE
unsigned emails

### ISSUE THREAD

**Issue by hengistthenord** [ckan/ckan#2862]
since our last upgrade to 2.4 we are adding users and see an issue I want some assist with .

 new users get an email - telling them their username and where to log in. but some don't get this email, they seem to be Hotmail/outlook users, on looking at the sent email it does not come from an email address, any suggestions as to where I might change this behaviour, I tried smtp.mail_from = mark@somewhere.com in the config but this is not fixing the issue, 

Im not sure whether this is base ckan issue or customuserpriviledges extension that's in play.
or whether its just Hotmail/outlook issue that I cant overcome

**Comment by wardi** [comment]
smtp.mail_from is the right setting. Are your non- Hotmail/outlook users getting mails with From addresses now?

**Comment by hengistthenord** [comment]
sadly no one is. 
the mail arrives at various other email accounts and corporate places with 
Data North Yorkshire <None> as the from not the email account I have set in the config parameter.

but most people are getting the email . the addresses are correct that we are sending to and no strange characters in the address

but we are surprised that any get received with the apparent lack of a from email address and surprised our mail server sends it without logging in with valid credentials. - need to think about that.

one of the penalties of being unsure and not knowing where in the chain of unfamiliar settings its going wrong what means that I did not have values for 
smtp.user or smtp.password

it didnt need it for testing but now we are live it seems that it might matter to some.

so i have changed that and put some values in and i now do get the email from the person i expect to and its no longer blank.

I hope that will fix the issue with the few who did not get their mail.
